mne/viz/tests/test_topomap.py

```diff
@@ -86,6 +86,7 @@ def test_plot_topomap():
     evoked.plot_topomap(times, ch_type='planar2', res=res)
     evoked.plot_topomap(times, ch_type='grad', mask=mask, res=res,
                         show_names=True, mask_params={'marker': 'x'})
+    evoked.plot_topomap("peaks")
```

Comment:
... makes sense. Maybe you want peaks returned for some other purpose, too. But - tomorrow :)

Any chances of getting #2104 merged this weekend?